src/quacc/runners/ase.py

```diff
@@ -362,3 +369,171 @@ def _set_sella_kwargs(self, optimizer_kwargs: dict[str, Any]) -> None:
+    max_steps : int or None, optional
+        Maximum number of steps to take.
+    optimizer : Optimizer or BFGS, optional
+        Optimizer class to use.
+    optimizer_kwargs : dict or None, optional
+        Dictionary of kwargs for the optimizer.
+    run_kwargs : dict or None, optional
+        Dictionary of kwargs for the run() method of the optimizer.
+    neb_kwargs : dict or None, optional
+        Dictionary of kwargs for the NEB.
+    copy_files : str or dict or None, optional
+        Files to copy before running the calculation.
+
+    Returns
+    -------
+    Optimizer
+        The ASE Optimizer object used in the NEB run.
+    """
+    # Copy atoms so we don't modify it in-place
+    images = copy_atoms(images)
+    neb = NEB(images, **neb_kwargs)
+
+    dir_lists = []
+    # Perform staging operations
+    # this calc_setup function is not suited for multiple Atoms objects
+    for image in images:
+        tmpdir_i, job_results_dir_i = calc_setup(image, copy_files=copy_files)
+        dir_lists.append([tmpdir_i, job_results_dir_i])
+
+    # Set defaults
+    optimizer_kwargs = recursive_dict_merge(
+        {
+            "logfile": "-" if SETTINGS.DEBUG else dir_lists[0][0] / "opt.log",
+            "restart": dir_lists[0][0] / "opt.json",
+        },
+        optimizer_kwargs,
+    )
+    run_kwargs = run_kwargs or {}
+
+    # Check if trajectory kwarg is specified
+    if "trajectory" in optimizer_kwargs:
+        msg = "Quacc does not support setting the `trajectory` kwarg."
+        raise ValueError(msg)
+
+    # Define the Trajectory object
+    traj_file = dir_lists[0][0] / "neb.traj"
+    traj = Trajectory(traj_file, "w", atoms=neb)
+    optimizer_kwargs["trajectory"] = traj
+
+    # Set volume relaxation constraints, if relevant
+    for i in range(len(images)):
+        if relax_cell and images[i].pbc.any():
+            images[i] = FrechetCellFilter(images[i])
+
+    # Run optimization
+    with traj, optimizer(neb, **optimizer_kwargs) as dyn:
+        dyn.run(fmax=fmax, steps=max_steps, **run_kwargs)
+
+    # Store the trajectory atoms
+    dyn.traj_atoms = read(traj_file, index=":")
+
+    # Perform cleanup operations
+    for ii, image in enumerate(images):
+        calc_cleanup(image, dir_lists[ii][0], dir_lists[ii][1])
+
+    return dyn
+
+
+def _geodesic_interpolate_wrapper(
+    reactant: Atoms,
+    product: Atoms,
+    nimages: int = 20,
+    perform_sweep: bool | None = None,
+    convergence_tolerance: float = 2e-3,
+    max_iterations: int = 15,
+    max_micro_iterations: int = 20,
+    morse_scaling: float = 1.7,
+    geometry_friction: float = 1e-2,
+    distance_cutoff: float = 3.0,
+) -> tuple[list[str], list[list[float]]]:
+    """
+    Interpolates between two geometries and optimizes the path with the geodesic method.
+
+    Parameters
+    ----------
+    reactant_product_atoms : List[Atoms]
+        List of ASE Atoms objects containing initial and final geometries.
```

Comment:
This is not matching your function signature.

Reply:
Fixed it earlier, probably. I could not find it now.